### ISSUE TITLE
Hello, I think some of the unreasonable about your test set settings

### ISSUE THREAD

**Issue by 22w2w2** [andabi/music-source-separation#12]
Your project is great, but I still have some confusion. In the testing phase, you still use _sample_range when reading the file, and np.random.choice in _sample_range. Is this not a bad idea? Because in the test we should test all the speech sequences, rather than randomly taking a starting position, very much looking forward to your answer

**Comment by ShengleiH** [comment]
@22w2w2 I agree with you, and I am confused about the bss_eval metric used here, there are music and vocal "gndsr, gsir, gsar", but I think only the vocal part is reasonable.
